test/e2e/storage/utils/utils.go

```diff
@@ -647,3 +648,28 @@ func CheckWriteToPath(pod *v1.Pod, volMode v1.PersistentVolumeMode, path string,
 	VerifyExecInPodSucceed(pod, fmt.Sprintf("echo %s | base64 -d | sha256sum", encoded))
 	VerifyExecInPodSucceed(pod, fmt.Sprintf("echo %s | base64 -d | dd of=%s bs=%d count=1", encoded, pathForVolMode, len))
 }
+
+// findMountPoints returns all mount points on given node under specified directory.
+func findMountPoints(nodeIP string, dir string) []string {
+	result, err := e2essh.SSH(fmt.Sprintf(`find %s -type d -exec mountpoint {} \; | grep 'is a mountpoint$'`, dir), nodeIP, framework.TestContext.Provider)
```

Comment:
Don't use ssh please, try HostExec:
https://github.com/kubernetes/kubernetes/blob/c1c0ba1c7681edcd38836c5c3a85abae7c8f3659/test/e2e/storage/utils/host_exec.go#L28-L32

Check its usage in other e2e tests.

Reply:
sure

Reply:
done